docs/Node/ReadLine.md

```diff
@@ -12,37 +12,85 @@ A handle to a console interface.
 
 A handle can be created with the `createInterface` function.
 
+#### `READLINE`
+
+``` purescript
+data READLINE :: !
+```
+
+The effect of interacting with a stream via an `Interface`
+
+#### `InterfaceOptions`
+
+``` purescript
+data InterfaceOptions
+```
+
+Options passed to `readline`'s `createInterface`
```

Comment:
Maybe add something to the docs here that explains that this is a phantom type, so that it's clear you never actually get a value of this type?